src/components/Layouts/Footer/Partials/index.jsx

```diff
@@ -1,5 +1,7 @@
 import UsefulLinks from "./usefulLinks";
 import Icons from "./footerIcons";
 import NewsItemList from "./newsItemList";
-
-export { NewsItemList, UsefulLinks, Icons };
+import newsList from "./constants";
```

Comment:
https://stackoverflow.com/questions/35665759/es6-how-can-you-export-an-imported-module-in-a-single-line

Please check the link above and convert the import/export to be one line for each partials :+1: